recipe/meta.yaml

```diff
@@ -28,12 +29,13 @@ source:
    # library paths, even if BUILD_SHARED_LIBS=1
    # This patch fixes that.
     - 0003-LEMONConfig.cmake.patch
+   # export symbols for shared lib on windows
+    - 0004-allow-symbols-to-be-exported-in-shard-libs-on-win.patch  # [win]
```

Comment:
I guess there is no way to submit these patches upstream easily. we should get on mailing list and submit there maybe

Reply:
oh wow, I was just checking and the website is back up - the issue tracker is active also, so there might be a way to sneak it in. In the meantime I'll merge this PR